### PR TITLE
Scissors-Paper-Stone (Part 1)

### PR DESCRIPTION
**Pull request by Melvin-Disastra**
Please fill out the survey before submitting the pull request. Thanks!

🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

How many hours did you spend on this assignment?
2 and a half ish hours
Please fill in one error and/or error message you received while working on this assignment.
This value is defined but not called anywhere else
What part of the assignment did you spend the most time on?
Looking for missing brackets when defining the rules
Comfort Level (1-5):
2
Completeness Level (1-5):
4
What did you think of this deliverable?
I got it working but I can tell my code is inefficient. I hope I actually can make it more efficient with further learning.
Is there anything in this code that you feel pleased about?
Once I solved it I can immediately picture alternative ways to do it

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,4 +1,80 @@
...
-};
+// variable assignments
+var scissors = "scissors";
+var paper = "paper";
+var stone = "stone";
+var reversedScissors = "reversed scissors";
+var reversedPaper = "reversed paper";
+var reversedStone = "reversed stone";
+var instructions =
+  'Type in "scissors" "paper" or "stone" to throw as your next hand. Good luck! <br>(Psst. You can add reversed before your hand to do inverted scissors papers stone!)';
```
Comment (emily-zhangg) [inline script.js:9]: I like the additional instructions placed here that makes the game much easier to be played

File: script.js [hunk 2]
```diff
@@ -1,4 +1,80 @@
...
+  ) {
+    return "You Win!";
+  }
+  if (
+    (userHand == scissors && computerHand == stone) ||
+    (userHand == paper && computerHand == scissors) ||
+    (userHand == stone && computerHand == paper) ||
+    (userHand == reversedPaper && computerHand == stone) ||
+    (userHand == reversedScissors && computerHand == paper) ||
+    (userHand == reversedStone && computerHand == scissors)
```
Comment (emily-zhangg) [inline script.js:49]: good job implementing || and && that you learnt in class in your code

File: script.js [hunk 3]
```diff
@@ -1,4 +1,80 @@
...
+}
+// looked for missing brackets here for half the assignment time
+// result message output, included with every successful loop
+var resultMessage = function (computerHand, userHand) {
+  return (
+    "The computer chose " + computerHand + ". <br> You chose " + userHand + "."
+  );
+};
+// error message for unexpected inputs
+var main = function (input) {
```
Comment (emily-zhangg) [inline script.js:62]: i like how you managed to pull out your logic intensive code to its own helper functions which makes the code more reusable and readable

**Review by yiqiingg — COMMENTED** [review]
great job melvin!!

**Pull request closed without merging**